Head tags

francescozaia opened the pull request:
- Added Social metatags, icons, favicons
- Added Analytics

Missing:
- Title and descriptions are hardcoded, they should change on page change.
- Analytics tracking ID should change according to the ENV, I left a comment with the live tracking ID.

francescozaia committed:
Added favicons, descriptions and analytics

xrmx commented on readthedocs/templates/docsitalia/includes/analytics.html:
```diff
@@ -0,0 +1,20 @@
+- UA-122854273-2: Live - https://docs.italia.it/
+{% endcomment %}
+
+{% with "UA-122854273-1" as trackingID %}
```
eh questo dobbiamo ricordarci di gestirlo da backend

francescozaia replied:
Sì infatti come sempre confidavo in un intervento per sistemare quanto indicato come "missing" nella descrizione della PR.

xrmx replied:
Idealmente sarebbe bello riciclare GLOBAL_ANALYTICS_CODE e DASHBOARD_ANALYTICS_CODE, vedi templates/base.html

xrmx reviewed (approved):
Occhio a splittare in file da poche righe perchè gli include non sono gratis

francescozaia commented:
Si può mettere tutto nel file `meta.html` senza alcun problema.

xrmx commented on readthedocs/templates/docsitalia/includes/head.html:
```diff
@@ -1,24 +1,16 @@
+{% include 'docsitalia/includes/favicons.html' %}
 
-<link rel="alternate" type="application/rss+xml" title="" href="feed.xml">
+<title>Docs Italia</title>
```
questo deve andare dentro ad un blocco

francescozaia committed:
- Rimossi include non necessari
- Gestione Titolo
- Utilizzo GLOBAL_ANALYTICS_CODE
- Rimozione codice inutilizzato

francescozaia commented:
- Ho aggiunto una migliore gestione del titolo di pagina, alla fine usando una soluzione simile a RTD perché poi altrimenti avrei dovuto modificare anche tutte le singole pagine
- Ho Utilizzato GLOBAL_ANALYTICS_CODE, ma rimane il fatto che è [impostato a `None`](https://github.com/italia/docs.italia.it/blob/italia-2.3.13/readthedocs/settings/base.py#L338), ma invece dovrebbe cambiare in base all'ambiente con i tracking ID descritti in un commento sopra lo script
- Rimosso un po' di codice inutilizzato (peraltro ho notato ci sono due file con codice duplicato: `/readthedocs/templates/docsitalia/overrides/base.html` e `/readthedocs/templates/docsitalia/base.html`, non si possono DRY?

xrmx commented:
Per GLOBAL_ANALYTICS_CODE va fatta PR sugli script di deploy, nel terzo punto hai messo due path uguali :)
